modules/apps/trash/trash-web/src/main/resources/META-INF/resources/js/TrashManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {postForm} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {restoreEntriesURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const deleteSelectedEntries = () => {
+		if (
+			confirm(
+				Liferay.Language.get('are-you-sure-you-want-to-delete-this')
+			)
+		) {
+			const form = document.getElementById(`${portletNamespace}fm`);
+
+			if (form) {
+				submitForm(form);
+			}
+		}
+	};
+
+	const restoreSelectedEntries = () => {
+		const form = document.getElementById(`${portletNamespace}fm`);
+
+		if (form) {
+			submitForm(form);
+		}
+		submitForm(form, restoreEntriesURL);
```

Comment:
This does not look right, the entry is both deleted and restored? Probably needs to be
```js
if (form) {
    submitForm(form, restoreEntriesURL);
}
```